aspnetcore/blazor/progressive-web-app/index.md

```diff
@@ -147,14 +147,36 @@ In the app's `wwwroot/index.html` file:
 
 :::moniker-end
 
-* Add the following `<script>` tag inside the closing `</body>` tag immediately after the `blazor.webassembly.js` script tag:
+:::moniker range=">= aspnetcore-10.0"
+
+* Add the following JavaScript inside the closing `</body>` tag immediately after the `blazor.webassembly.js` script tag:
+
+  ```html
+  <script>
+    navigator.serviceWorker.register('service-worker.js', { updateViaCache: 'none' });
```

Comment:
This change isn't .NET 10 specific. It's just that in .NET 10 we include in the template, but it's a good thing to do regardless of .NET version